pkg/translator/prometheusremotewrite/metrics_to_prw_v2.go

```diff
@@ -105,7 +105,12 @@ func (c *prometheusConverterV2) fromMetrics(md pmetric.Metrics, settings Setting
 				case pmetric.MetricTypeExponentialHistogram:
 					// TODO implement
 				case pmetric.MetricTypeSummary:
-					// TODO implement
+					dataPoints := metric.Summary().DataPoints()
+					if dataPoints.Len() == 0 {
+						errs = multierr.Append(errs, fmt.Errorf("empty data points. %s is dropped", metric.Name()))
```

Comment:
We don't append errors if other data point types are empty.  Why are we doing that here?

Reply:
fixed.